townhall/myapi/dao.py

```diff
@@ -387,3 +387,7 @@ def get_project(id: int) -> typing.Optional[Project]:
             return project
         except Project.DoesNotExist:
             return None
+
+    @staticmethod
+    def get_project_all() -> typing.Optional[Project]:
```

Comment:
[all() returns a queryset](https://docs.djangoproject.com/en/5.1/ref/models/querysets/#django.db.models.query.QuerySet.all), so the type hinting here should be  `-> QuerySet[Project]`